spec/user_interface_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'user_interface'
+
+RSpec.describe UserInterface do
+  let(:error_message) { described_class::ERROR_MESSAGE }
+  let(:output) { StringIO.new }
+  let(:valid_input) { StringIO.new('1') }
+
+  it 'Prints menu of options for user to choose' do
+    ui = described_class.new(valid_input, output)
+
+    ui.run
+
+    expect(output.string).to include(described_class::MENU_MESSAGE)
+  end
+
+  it 'Clears the screen before printing the menu' do
+    ui = described_class.new(valid_input, output)
+
+    ui.run
+
+    expect(output.string).to include("\033[H\033[2J")
```

Comment:
Since the test says "before", implying order is important, you could expect both in the right order:

```ruby
    expect(output.string).to eq("\033[H\033[2J" + described_class::MENU_MESSAGE)
```